health/micro-ui/web/micro-ui-internals/packages/modules/health-dss/src/components/MetricChart.js

```
@@ -0,0 +1,154 @@
+  const { isLoading, data: response } = Digit.Hooks.dss.useGetChart({
+    key: id,
+    type: chartType,
+    tenantId,
+    requestDate: { ...value?.requestDate, startDate: value?.range?.startDate?.getTime(), endDate: value?.range?.endDate?.getTime() },
+    filters: {...value?.filters, projectTypeId: selectedProjectTypeId},
+    moduleLevel: value?.moduleLevel
+  });
+
+  useEffect(() => {
+    if (response) {
+      let plots = response?.responseData?.data?.[0]?.plots || null;
```

Biome (1.9.4)

[error] This let declares a variable that is only assigned once.

'plots' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)